src/gatherCards.js

```diff
@@ -20,7 +20,13 @@ function loopGather(maxFollow, maxPeople, resolve, reject, index = 0, cards = []
 		for(; index < scrolledPeople.length && cards.length < maxFollow; index++){
 			let card = createCard(scrolledPeople[index]);
 			// Score the card here
-			cards.push(card);
+			if (card.checkForFollowed || card.isProtected) {
```

Comment:
At some point we can swap this entire if statement out for a function that gets passed in. That will make gatherCards more versatile and work for more than just following.